privacy-protections/referrer-trimming/main.js

```diff
@@ -22,7 +22,7 @@ function generateNavigationTest(url) {
 
     if (localStorage[key]) {// test already finished before
         return JSON.parse(localStorage[key]);
-    } else if(currentURL.searchParams.get('js')) {// test finished now
+    } else if(currentURL.searchParams.get('js') !== null) {// test finished now
```

Comment:
unrelated fix for referrer trimming - we have to test for null specifically because Brave, in certain scenarios trims referrer to an empty string.